src/server/actions/Volunteer.ts

```diff
@@ -0,0 +1,17 @@
+import { VolunteerEntity } from '@/types/dataModel/volunteer';
+import VolunteerSchema from '@/server/models/Volunteer';
+import dbConnect from '@/utils/db-connect';
+
+/**
+ * Gets all volunteers.
+ * @returns Collection of VolunteerEntities in the database, or null if there are none.
+ */
+export async function getAllVolunteers(): Promise<VolunteerEntity[] | null> {
+  try {
+    await dbConnect();
+    const volunteers: VolunteerEntity[] | null = await VolunteerSchema.find();
```

Comment:
This should return a `VolunteerResponse` array, where the `previousOrganization` field is populated. Accomplish this by doing `.find().populate('previousOrganization')`. I tested this locally, and it is not quite working for some reason. Let's take a look after today's meeting